content/docs/command-reference/remove.md

```diff
@@ -13,35 +14,32 @@ positional arguments:
 
 ## Description
 
-This command safely removes stages from
-[dvc.yaml](/doc/user-guide/dvc-files-and-directories#dvcyaml-file), their
-`.gitignore` entries, and optionally removes from the <abbr>workspace</abbr>
-files or directories that are tracked by DVC. It takes one or more stage names
-(see `-n` option of `dvc run`) or `.dvc` files as target, removes it, and
-optionally removes all of its outputs (`outs` field).
+Safely removes `.dvc` files or stages from `dvc.yaml`. This includes deleting
```

Comment:
I like the previous version better (even though it was not exactly correct with regards to `.dvc` files). We need to find a better language -  something that unites stages and .dvc - a higher level abstraction to convey the essence of this command.

Reply:
> need to find a better language - something that unites stages and .dvc

We have metafiles for now (DVC Files soon) but it doesn't really apply here since the command's behavior is different for one or the other. I think that we need to specify each one.